odl/solvers/functional/default_functionals.py

```diff
@@ -1326,6 +1376,185 @@ def convex_conj(self):
                                  constant=constant)
 
 
+class NuclearNorm(Functional):
+
+    """The nuclear norm.
+
+    See article:
+    Collaborative Total Variation: A General Framework for Vectorial TV Models
```

Comment:
J. Duran, M. Moeller, C. Sbert, and D. Cremers. Collaborative Total Variation: A General Framework for Vectorial TV Models, SIAM Journal of Imaging Sciences 9(1): 116--151, 2016.